src/Tools/dotnet-user-secrets/src/dotnet-user-secrets.csproj

```diff
@@ -18,6 +18,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <Reference Include="Newtonsoft.Json" />
```

Comment:
Why does this project need to change?

Reply:
It was getting Newtonsoft.Json transitively before.

Reply:
https://github.com/aspnet/Extensions/pull/1028

Reply:
👍